cluster/images/etcd/Makefile

```diff
@@ -83,7 +83,7 @@ endif
 # This option is for running docker manifest command
 export DOCKER_CLI_EXPERIMENTAL := enabled
 # golang version should match the golang version of the official build from https://github.com/etcd-io/etcd/releases.
-GOLANG_VERSION?=1.16.15
+GOLANG_VERSION?=1.17.13
```

Comment:
This change invalidates comment above. Etcd v3.5.5+ uses go1.16.15. Could we update how new golang version should be picked?

Reply:
yep adding notes in dependencies.yaml